docs/reference/objects/product/promotion.md

```diff
@@ -8,33 +8,50 @@ When using a `promotion` object you have access to the following attributes
 
 # promotion.active
 
-Returns true if the promotion is currently active
+Returns true if the promotion is currently active.
+
+# promotion.category
+
+Returns the category of the promotion.
```

Comment:
Helpful to note that this will be either "Early release", "Special offer", or "Last minute" in case the theme developer needs to match this (can do so without trial and error).

Reply:
Nice one. Will add.